containerd-shim-v2/stream.go

```diff
@@ -5,11 +5,119 @@
 package containerdshim
 
 import (
+	"context"
 	"io"
+	"sync"
+	"syscall"
+
+	"github.com/containerd/fifo"
+)
+
+var (
+	bufPool = sync.Pool{
+		New: func() interface{} {
+			buffer := make([]byte, 32<<10)
```

Comment:
Please makes this a constant, and explain why it is needed.

Reply:
Done